revapi-java-spi/src/main/java/org/revapi/java/spi/Code.java

```diff
@@ -104,7 +104,7 @@
         POTENTIALLY_BREAKING, "annotationType"),
     ANNOTATION_NOW_INHERITED("java.annotation.nowInherited", NON_BREAKING, NON_BREAKING, POTENTIALLY_BREAKING,
             "annotationType"),
-    ANNOTATION_NO_LONGER_PRESENT("java.annotation.noLongerPresent", BREAKING, NON_BREAKING, POTENTIALLY_BREAKING),
+    ANNOTATION_NO_LONGER_PRESENT("java.annotation.noLongerPresent", BREAKING, BREAKING, POTENTIALLY_BREAKING),
```

Comment:
Should this last arg now be `null`? Just guessing based on some other examples in here

Reply:
You're right that it does always seem to be `null` when the binary level is `BREAKING`. It's `semanticSeverity`, which we ignore. I don't totally know what the best choice is, so my inclination is to leave the code as is for now and ask this question when I PR the upstream repo.